StigData/Processed/WindowsDefender-All-1.6.org.default.xml

```diff
@@ -5,19 +5,19 @@
     Each setting in this file is linked by STIG ID and the valid range is in an
     associated comment.
 -->
-<OrganizationalSettings fullversion="1.4">
+<OrganizationalSettings fullversion="1.6">
   <!-- Ensure 'V-75237' -ge '0' -and 'V-75237' -le '7'-->
-  <OrganizationalSetting id="V-75237" ValueData="0" />
+  <OrganizationalSetting id="V-75237" ValueData="" />
```

Comment:
These values should be populated, review the previous org file as a reference.

Reply:
Same

Reply:
Fixed